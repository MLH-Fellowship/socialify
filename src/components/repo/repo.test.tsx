import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Repo from './repo'

test('Repo renders', () => {
  const repo = shallow(<Repo />)

  expect(toJson(repo)).toMatchSnapshot()
  expect(repo.hasClass('repo-wrapper')).toBe(true)
  expect(repo.find('.repo-input-container').length).toBe(1)
})
