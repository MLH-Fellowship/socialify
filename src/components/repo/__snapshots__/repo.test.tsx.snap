// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo renders 1`] = `
<div
  className="repo-wrapper"
>
  <form
    className="repo-input-container"
    onSubmit={[Function]}
  >
    <Input
      autoFocus={true}
      onChange={[Function]}
      placeholder="Enter GitHub repo"
      prefix={<ForwardRef(GithubOutlined) />}
      suffix={
        <ForwardRef(InternalButton)
          block={false}
          ghost={false}
          htmlType="submit"
          loading={false}
          type="primary"
        >
          Submit
        </ForwardRef(InternalButton)>
      }
      type="text"
      value=""
    />
  </form>
</div>
`;
